demo/src/main/resources/index.html

```diff
@@ -6,9 +6,9 @@
     <link rel="stylesheet" type="text/css" href="css/github-gist.css">
     <!--script type="text/javascript" src="js/lunr.min.js"></script-->
     <!--script type="text/javascript" src="js/index.js"></script-->
-    <script type="text/javascript" src="js/demo.js"></script>
+    <script type="text/javascript" src="js/demo.js" defer></script>
     <script src="js/highlight.pack.js"></script>
 </head>
 <!--body onload="demo.LunrDemo().loadIndex(index);" class="uu"-->
-<body onload="demo.BootstrapDemo().main();"></body>
```

Comment:
Bundled artifacts do not export an entry point, they are self-executed when the script is loaded. I added the `defer` attribute (line 9) to ensure that the DOM is effectively loaded before the script is executed.